src/ssh.h

```diff
@@ -0,0 +1,25 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+#include <memory>
+
+#include <QString>
+
+class QStringList;
+class QProcess;
+
+// used in the dialog to test connection and copy ssh keys
```

Comment:
can you please expand the documentation? what is the difference between options, command and executable?